contrib/galaxy.fedora.uwsgi.handlers-init

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+#
+# Init file for launching Galaxy with uwsgi and multiple job handlers (http://galaxyproject.org/)
+#   Suitable for use on Fedora and derivatives (RedHat Enterprise Linux, Scientific Linux, CentOS)
+#
+# Contributed by Brad Chapman
+# Improved by Remi Marenco
+#
+# chkconfig: 2345 98 20
+# description: Galaxy http://galaxyproject.org/
+
+#--- config
+
+SERVICE_NAME="galaxy"
+RUN_AS="galaxy"
+RUN_IN="/path/to/galaxy-dist"
+
+#--- main actions
+
+start() {
+	echo "Starting $SERVICE_NAME... "
+	cmd_uwsgi="cd $RUN_IN && PYTHONPATH=eggs/PasteDeploy-1.5.0-py2.7.egg uwsgi --ini-paste config/galaxy.ini --master-fifo masterFifo &"
+        cmd_galaxy_daemons="cd $RUN_IN && GALAXY_RUN_ALL=1 sh run.sh --daemon"
+	case "$(id -un)" in
+		$RUN_AS)
+			eval "$cmd_uwsgi"
+			eval "$cmd_galaxy_daemons"
+			;;
+		root)
```

Comment:
You don't probably need a special case for `$RUN_AS`, you can use the same commands below.